benchmarks/distributed/lambda_runtimes/cpp_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Create CMake project
+cmake_minimum_required(VERSION 3.9)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+#project(test-python LANGUAGES CXX)
+set(LAMBDA_NAME test-python)
+
+# Create AWS cpp Project
+find_package(aws-lambda-runtime)
+#find_package(AWSSDK COMPONENTS s3)
+add_executable(${LAMBDA_NAME} "main.cpp" "lambda_main.cc" "sighandler.cc")
+target_link_libraries(${LAMBDA_NAME} PRIVATE AWS::aws-lambda-runtime ${AWSSDK_LINK_LIBRARIES})
+
+# Link Python libraries + interpreter
+set(PYTHON_RESOURCES_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../python38_resources)
+#target_include_directories(${LAMBDA_NAME} PRIVATE ${PYTHON_RESOURCES_DIR}/include/python3.7m/)
+target_include_directories(${LAMBDA_NAME} PRIVATE /usr/include/python3.8/)
+
+#find_library(LAMBDA_PYTHON_PACKAGE NAMES python3.8) #PATHS ${PYTHON_RESOURCES_DIR}/libpy38 REQUIRED)
+##find_library(LAMBDA_PYTHON_PACKAGE NAMES python3.7m PATHS ${PYTHON_RESOURCES_DIR}/lib REQUIRED)
+#message("(cpp_runtime) Found Lambda Python package: ${LAMBDA_PYTHON_PACKAGE}")
+##link_directories(${CMAKE_SOURCE_DIR}/lib)
+#target_link_libraries(${LAMBDA_NAME} PRIVATE ${LAMBDA_PYTHON_PACKAGE})
+
+# link tuplex - use the libraries from awslambda
+target_link_libraries(${LAMBDA_NAME} PRIVATE libio libutils libcore ${Boost_LIBRARIES} ${LibMagic_LIBRARIES})
+
+# Set compile flags/features
+target_compile_features(${LAMBDA_NAME} PRIVATE "cxx_std_11")
```

Comment:
shouldn't this be done via C++14 above? seems redundant.

Reply:
i'm not entirely sure - https://stackoverflow.com/questions/49012222/c11-activation-with-target-compile-feature-or-set-cmake-cxx-standard/49027994. should i prefer one over the other?